### ISSUE TITLE
The model does not overfit

### ISSUE THREAD

**Issue by pourfard** [lizhengwei1992/Semantic_Human_Matting#18]
Hi,

I decreased the size of training images to just 20 images, but after training tnet loss remains around 0.3. Is this normal? I thought that it should reach around 0 after a few epochs?

**Comment by tobechao** [comment]
@pourfard hi, pourfard, I train the t-net overfit, loss also about 0.3,  have you solved?
